todo/src/actions/index.js

```diff
@@ -0,0 +1,13 @@
+export const addTodo = (todo) => {
+    return {
+        type: 'ADD_TODO',
```

Comment:
For your actions and reducers, set up constants to use for the type (i.e. `export const ADD_TODO = "ADD_TODO"`). That way your compiler will throw an error if you have a typo when referring to the variable.